two-step-view/README.md

```diff
@@ -0,0 +1,156 @@
+# Two-Step View
```

Comment:
Add front-matter as instructed in https://github.com/iluwatar/java-design-patterns/wiki/21.-New-Pattern-Template